v8/Client/ParallelServiceProxy.cs

```diff
@@ -301,13 +301,19 @@ public IEnumerable<DiscoveryResponse> Execute(IEnumerable<DiscoveryRequest> requ
     /// </remarks>
     public class ParallelOrganizationServiceProxy : ParallelServiceProxy<OrganizationServiceManager>
     {
+        private const int RateLimitExceededErrorCode = -2147015902;
+        private const int TimeLimitExceededErrorCode = -2147015903;
+        private const int ConcurrencyLimitExceededErrorCode = -2147015898;
+
+        private const int MaxRetries = 3;

```

Comment:
Make this configurable, part of OrganizationServiceProxyOptions, set default to 3 if none provided